### DISCUSSION TITLE
Is it possible to send _notifier from the service?

### DISCUSSION THREAD

**Discussion by vitalybrandes** [OrchardCMS/OrchardCore#13692] [Q&A] [answered]
Is it possible to send _notifier from the service (code)

**Comment by hishamco** [comment]
Yes, you can use `INotifier` from DI, then call

```csharp
await _notifier.SuccessAsync(H["Your message."]);
```

where `H` is `IHtmlLocalizer`

**Reply by vitalybrandes** [reply]
> Yes, you can use `INotifier` from DI, then call
> 
> ```cs
> await _notifier.SuccessAsync(H["Your message."]);
> ```
> 
> where `H` is `IHtmlLocalizer`

I want to use it in my own service and not in controller or page.
I am trying to get any data from external api, and I would like to notify the response status , from the service responsible for this process.

**Reply by hishamco** [reply, marked as answer]
While `INotifier` registered in DI you can use it from your own service too
